cmd/witness/cmd/run_test.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2021 The Witness Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+	"testing"
+
+	"github.com/testifysec/witness/cmd/witness/options"
+	"github.com/testifysec/witness/pkg/cryptoutil"
+)
+
+func Test_runRunRSAKeyPair(t *testing.T) {
+
+	priv, _ := rsakeypair(t)
+	keyOptions := options.KeyOptions{
+		KeyPath: priv.Name(),
+	}
+
+	workingDir := t.TempDir()
+
+	runOptions := options.RunOptions{
+		KeyOptions:   keyOptions,
+		WorkingDir:   workingDir,
+		Attestations: []string{},
+		OutFilePath:  workingDir + "outfile.txt",
```

Comment:
this is a test so i don't wanna nitpick too much, but we should prefer to use `[filepath.Join](https://github.com/testifysec/witness/blob/e194af87ef8474f9f333b0075ed9c7bb83570c10/cmd/witness/cmd/root.go#L52)` for these types of functions to make sure directory separators, etc, are handled properly